app/services/discussion_service.rb

```diff
@@ -28,6 +28,7 @@ def self.create(discussion:, actor:)
     return false unless discussion.valid?
 
     discussion.save!
+    DiscussionReader.for(discussion: discussion, user: actor).set_volume! :loud
```

Comment:
Hm, we already do `set_volume_as_required!` on discussion create, I think we can just change that to being loud and take this out of the service

```
  config.listen('discussion_create') do |discussion, actor|
    DiscussionReader.for(discussion, actor).set_volume!(:loud)
  end

  config.listen('discussion_update', 'comment_like') do |model, actor|
    DiscussionReader.for_model(model, actor).set_volume_as_required!
  end
```